shared/charts/gpii-preferences/templates/deployment.yaml

```diff
@@ -9,7 +9,6 @@ spec:
     rollingUpdate:
       maxSurge: {{ .Values.rollingUpdate.maxSurge }}
       maxUnavailable: {{ .Values.rollingUpdate.maxUnavailable }}
-    type: RollingUpdate
```

Comment:
Why is this removed? Probably doesn't make any difference as `RollingUpdate` is the default... but?

Reply:
This is exactly why it is removed :) It is default type and it was present in preferences chart and missing in flowmanager chart, so I removed it for consistency. I could add it into flowmanager chart if you prefer that.

Reply:
Ok, thanks for the explanation, I think it's fine. Consistency is good 👍